### DISCUSSION TITLE
Discord Mention

### DISCUSSION THREAD

**Discussion by punksinatra** [mbround18/valheim-docker#299] [Ideas]
Would it be possible to mention a role with the discord webhook? I help run a gaming discord server and I think it will be useful to have the webhook mention a role (i.e. valheim or valheim players) whenever the server restarts, shutdown, backup, or there are events etc. 

If it's already possible can someone point me the right direction I check the docs and did not find any information. I also searched discussions but did not find one (I maybe blind)

Thanks

**Comment by mbround18** [comment]
Yes you should be able to mention a role, use the env variables in [webhook docs](https://github.com/mbround18/valheim-docker/blob/main/docs/webhooks.md) to include `<@ROLE_ID>`. Where ROLE_ID is, go into discord -> server settings -> roles -> right click the role -> Copy Id

**Reply by punksinatra** [reply]
This is the only variable I am seeing in the webhook docs
![image](https://user-images.githubusercontent.com/22606153/112511608-8c80de80-8dcd-11eb-87ca-12c30ac140f6.png)

**Reply by punksinatra** [reply]
Clicking on raw shows the full information

https://raw.githubusercontent.com/mbround18/valheim-docker/main/docs/webhooks.md

**Reply by mbround18** [reply]
I am unsure how those got commented :P 

Use the variables described in the raw format to add a text body. In the future, I aim to make this a configuration file for easier editing.
